src/app/dashboard/filter-ops/filter-ops.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="filter-ops">
+  <app-load-title title="{{ title }}" [isLoading]="isLoading"></app-load-title>
+
+  <ul class="filter" (clickOutside)="hide()">
+    <li class="dropdown arrow-right">
+      <a (click)="toggle()" [ngClass]="{ active: anyValueSet() }">{{ 'filter' | translate }}</a>
+      <a (click)="clearParams()">{{ 'reset' | translate }}</a>
+      <div class="dropdown-wrapper" [ngClass]="{ hidden: !showing }">
+        <div class="dropdown-content">
+          <div class="filter-grid">
+            <ng-container *ngFor="let filter of conf">
+              <span class="start-row filter-cell" (click)="showParams()">
+                {{ filter.label }}
+              </span>
+              <span
+                *ngFor="let op of filter.options; index as i"
+                [ngClass]="{
+                  'continue-row-med': (i + 1) % 5 === 1,
+                  'end-row': i + 1 === filter.options.length
+                }"
+                class="filter-cell"
+                ><a
+                  (click)="toggleParamValue(filter.name, op, filter.multi)"
+                  *ngIf="!op.input"
+                  [ngClass]="{ active: valueIsSet(filter.name, op.value) }"
+                  class="toggle"
+                >
+                  {{ op.value }}
+                </a>
+                <label
+                  *ngIf="op.input"
+                  [ngClass]="{ active: valueIsSet(filter.name, inputRef.value, i) }"
+                  class="input-label"
+                >
+                  {{ op.value }}:
+                  <input
+                    #inputRef
+                    id="{{ op.input.id }}"
+                    type="{{ op.input.type }}"
```

Comment:
This is native behaviour for html date inputs - the max and min apply only to the calendar pop up.  To prevent exceeding the min/max settings with the keyboard would require the disabling of the keyboard.